content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -1,99 +1,195 @@
+So, in this case, since the customer is ready to pay the remaining amount, the :guilabel:`Regular
+Invoice` option should be selected, and the :guilabel:`Deduct down payments` option should *stay*
+selected. Then, click :guilabel:`Create and View Invoice`.
 
-.. image:: down_payment/down_payment_3.png
+Doing so reveals a separate :guilabel:`Invoices` page, listing all the invoices for that specific
+sales order. Each invoice line item displays all the necessary information related to each invoice,
+along with the current :guilabel:`Payment Status`.
+
+.. image:: down_payment/down-payment-invoices-page.png
    :align: center
-   :alt: What is a basic sales flow using down payments on Odoo Sales (2)?
+   :alt: The down payment invoices page in Odoo Sales.
+
+To complete the flow, select the invoice with the :guilabel:`Not Paid` status, which opens that
+invoice. On the invoice page, click :guilabel:`Confirm`, then :guilabel:`Register Payment`, and
+confirm that the remaining :guilabel:`Amount` is correct. Then, click :guilabel:`Create Payment`.
 
-To complete the flow, when the customer wants to pay the rest of his sales order, you must create
-another invoice. Once again, you will have the choice to make another down payment or to deduct all
-the down payments and so, paying the rest of the invoice as a regular invoice.
+That pop-up disappears, and a green `Paid` banner is now present on the invoice. The line items on
+the invoice show that the total amount has been paid, including mentions of the down payment, and
+the flow is complete.
 
-.. image:: down_payment/down_payment_4.png
+.. image:: down_payment/invoice-complete-paid-banner-2.png
    :align: center
-   :alt: What is a basic sales flow using down payments on Odoo Sales (3)?
+   :alt: The completed step of the down payment sales flow process in Odoo Sales.
 
-This flow is also possible with a down payment taking into account a fixed amount.
+This flow is also possible with the :guilabel:`Fixed amount` down payment option.
 
 .. important::
-   Be careful that if you do a down payment with a product using **delivered quantities** as
-   invoicing policy, you won’t be able to deduct all the down payments when it comes to invoicing
-   your customer. Indeed, you have to deliver a product before creating the final invoice. If
-   nothing has been delivered, you create a **credit note** that cancels the draft invoice
-   created after the down payment. To do so, you have to install the **Inventory App** to confirm
-   the delivery. Otherwise, you can enter the delivered quantity manually on the sales order.
+   If a down payment is used with a product that has a :guilabel:`Delivered quantities` invoicing
+   policy, all the down payments **won't** be able to be deducted when it comes time to invoice the
+   customer. This is because, due to the invoicing policy, the product(s) would have to be
+   delivered *before* creating the final invoice.
```

Comment:
100th char, contraction, wording
```suggestion
   policy, the down payments will **not** be able to be deducted when it comes time to invoice the
   customer. This is because, due to the invoicing policy, the product(s) would have to be delivered
   *before* creating the final invoice.
```